tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/base.tsx

```diff
@@ -256,6 +256,7 @@ export function FeatureShortTextInput(
       value={value}
       maxLength={serverData?.maximum_length}
       onChange={handleSetValue}
+      expensive
```

Comment:
We have onblur now, if it makes more sense to use that instead of delaying onchange